content/grants/chess-bounty.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Chess Improvements"
+date = "2022-08-22"
+
+[taxonomies]
+grant_type = [ "Bounty" ]
+grant_category = [ "App Dev" ]
+
+[extra]
+image = ""
+description = "Improvements for the Chess app to create a Chess experience comparable to that of any modern Chess website"
+reward = "3 stars"
+assignee = [""]
+grant_id = "B0175"
+champion = "~finmep-lanteb"
+completed = false
+work_request_link = "https://airtable.com/shr4qt9t9kz7RaOIa?prefill_Grant+ID=B0175&prefill_Grant+Name=Chess%3A%20Improved%2C%20Timed%2C%20%26%20Rated"
++++
+
+# Rationale
+~finmep-lanteb is sponsoring the improvement of the Chess app to create a Chess experience comparable to that of any modern Chess website. The original development of Chess on urbit functions in the context of game logic and ability to play with friends, but lacks certain features, such as notifications of your opponent completing their turn, which are necessary to present a complete app experience. These quality of life improvements offer a great opportunity for a team of 3 novice Urbit developers (preferably three Hoon School Live and App School Live graduates) to work together to complete this bounty, each earning a star in the process.
```

Comment:
"~finmep-lanteb is sponsoring the improvement of the Chess app to create an Urbit Chess experience comparable to that of any modern Chess website. The current Chess app meets the basic requirements to play a game with a friend, but lacks various quality of life features: move notifications, undo requests, `%pals` integration, etc. In addition, only unrated, correspondence games are currently possible. These missing features & improvements offer a great opportunity for a team of 3 novice Urbit developers (preferably three Hoon School Live and App School Live graduates) to work together to complete this bounty, each earning a star in the process."